### ISSUE TITLE
UI enhancement of Dashboard

### ISSUE THREAD

**Issue by rishuyadav** [ACM-VIT/gitpositive#18]
Need to enhance the UI according to the proposed App Design
- [ ]  Change the title color as per the design(using 2 colors) size to be kept the same as previous.
- [ ]  Change Button(Followers, Repositories, and Following) design for data to be displayed as in the below design

![image](https://user-images.githubusercontent.com/72988817/135718757-ada8cca0-67d0-4a43-aed7-97aafa073b6c.png)

**Comment by alok27a** [comment]
@gaganmalvi Can u pls assign this issue to me?

**Comment by vishal206** [comment]
Hi @rishuyadav, I would like to work on this issue. Can you assign me?

**Comment by alok27a** [comment]
It has been 2 days is this issue resolved?

**Comment by rishuyadav** [comment]
Hey, @vishal206 are you working on the issue, pls reply to lets us know or we can proceed with assigning it to someone else interested?

**Comment by vishal206** [comment]
yes, I'm working on it. Actually I'm not able to build the project in android studio.I have even posted a msg related to this in your discord server but I haven't got any help

**Comment by gaganmalvi** [comment]
Where in the Discord server? Could you tag me at @malviscape

**Comment by vishal206** [comment]
In Android channel ( in forktober'21 server). Am I on the correct server!. Bcos I'm not able to find your name @gaganmalvi

**Comment by gaganmalvi** [comment]
What's your error, please post here

**Comment by vishal206** [comment]
![image](https://user-images.githubusercontent.com/54732763/135984493-68687eb9-9796-41d0-b9c1-279b51001eb1.png)

"Unrecognized Android Studio (or Android Support plugin for IntelliJ IDEA) version '202.7660.26.42.7486908', please retry with version 2020.3.1 or newer."

this is the error I'm getting @gaganmalvi

**Comment by rishuyadav** [comment]
> ![image](https://user-images.githubusercontent.com/54732763/135984493-68687eb9-9796-41d0-b9c1-279b51001eb1.png)
> 
> "Unrecognized Android Studio (or Android Support plugin for IntelliJ IDEA) version '202.7660.26.42.7486908', please retry with version 2020.3.1 or newer."
> 
> this is the error I'm getting @gaganmalvi

Hey @vishal206 Go through this, hope it helps 
https://stackoverflow.com/questions/68215266/unrecognized-android-studio-version

**Comment by rishuyadav** [comment]
Did it help @vishal206 ?
Are u using Android Studio Arctic fox?

**Comment by vishal206** [comment]
Hey, it's working now.
I had to update my  android studio and SDK 
@rishuyadav 
> Did it help @vishal206 ? Are u using Android Studio Arctic fox?

**Comment by vishal206** [comment]
Task 1 about title colour, It's already in green and white. So what should I do about it?

**Comment by SiddyDevelops** [comment]
@rishuyadav  @gaganmalvi Please review my PR. Thank you
